### ISSUE TITLE
Normalization in Spatial Information Calculation

### ISSUE THREAD

**Issue by MasahiroNakano** [TomGeorge1234/SIMPL#7]
@TomGeorge1234 I think it's time to do this!

### Current implementation
The current implementation of spatial information is as follows.

```
if F is not None:
    F_pdf = (F+1e-6) / jnp.sum(F, axis=1)[:,None] # normalise the place fields
    I_F = jnp.sum(F_pdf * jnp.log(F_pdf), axis=1) # spatial information of the place fields
    metrics['information'] = I_F
```

This code normalizes the place fields with the **sum** of the activity, effectively computing a **negative entropy**.

### Issue
The standard way introduced in Skaggs, McNaughton, and Gothard (1992) (_An Information-Theoretic Approach to Deciphering the Hippocampal Code_) normalizes using the **mean** of the activity rather than the sum. The correct formula should be:

$I=\int_x \lambda(x) \log _2 \frac{\lambda(x)}{\lambda} p(x) d x$

### Proposed Changes
1. Clarify the metric name
The current implementation effectively computes a negative entropy rather than standard spatial information. To avoid confusion, renaming information to `negative_entropy` would be more appropriate.

2. Implement the standard spatial information measure
A separate implementation for `spatial_information` should follow the original Skaggs et al. definition, using mean-normalization.
This is a bit more tricky, as it requires modifying Kalmax as well. [`kalmax.kde.kde`](https://github.com/TomGeorge1234/KalMax/blob/main/kalmax/kde.py) needs to return the spatial occupancy together with `kernel_density_estimate`

```
if F is not None:
    # X is the spatial occupancy
    X_pdf = (X+1e-6) / jnp.mean(X, axis=1)[:,None] # normalise the spatial occupancy
    F_pdf = (F+1e-6) / jnp.mean(F, axis=1)[:,None] # normalise the place fields
    I_F = jnp.mean(F_pdf * jnp.log(F_pdf) * , axis=1) # spatial information of the place fields
    metrics['information'] = I_F
```

Would love to hear any thoughts or alternative suggestions on this! 🚀

**Comment by MasahiroNakano** [comment]
Assuming spatial occupancy `position_d being` (N_bins, )

```
# F = results.F.sel(epoch=epoch).values
# F = F.reshape(F.shape[0], -1)
# Convert to firing rate (spike per sec)
F = F / DT
F_mean = jnp.mean(F, axis=1)[:, None]
# normalise the spatial occupancy
X_pdf = position_d / jnp.sum(position_d)
# normalise the place fields
F_pdf = (F+1e-6) / F_mean
# spatial information of the place fields
I_F = (F_pdf * jnp.log(F_pdf)) @ X_pdf
```

I think this is the correct implementation.

**Comment by MasahiroNakano** [comment]
The code above should be the correct implementation for spatial information!
